core/unit_test/CMakeLists.txt

```diff
@@ -706,6 +724,12 @@ if(Kokkos_ENABLE_SERIAL)
     UnitTestMainInit.cpp
     ${Serial_SOURCES2}
```

Comment:
Why don't you list the supported backends and do:
```cmake
set(ViewSupport_backends SERIAL OPENMP ...)
foreach(backend)
    if(support backend)
        add_test
    endif()
endfunction()
```

Reply:
I moved it right up where we set the source files. The only thing is that the original separation (The way other tests are doing it) was there to do backend specific exclusions without having all the backend specific conditions in the initial list of tests.